internal/commands/accesslist/create_test.go

```diff
@@ -0,0 +1,120 @@
+package accesslist
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+
+	"github.com/Netflix/go-expect"
+)
+
+func TestCreateInputs(t *testing.T) {
+	for _, tc := range []struct {
+		description string
+		inputs      createInputs
+		procedure   func(c *expect.Console)
+		test        func(t *testing.T, i createInputs)
+	}{
+		{
+			description: "Should not prompt for inputs when flags provide the data",
```

Comment:
we're avoiding capitalization and punctuation in test descriptions, make this `"should not prompt..." (and similar adjustments for the others in this file)

(you'll see plenty of instances where this pattern is broken, it was instituted a bit late in the game.  but it's what we're doing going forward)

---

also, can we split this test table out into at least two variants: one for interactivity and one for not?

the non-interactive tests can be written so much simpler that its worth keeping them separate from the ones where we need the fancy pseudo-terminal